src/server.h

```diff
@@ -222,6 +222,12 @@ extern int configOOMScoreAdjValuesDefaults[CONFIG_OOM_COUNT];
 #define AOF_ON 1              /* AOF is on */
 #define AOF_WAIT_REWRITE 2    /* AOF waits rewrite to start appending */
 
+/* AOF return values for loadAppendOnlyFile() */
+#define AOF_OK 0
+#define AOF_EMPTY 1
+#define AOF_OPEN_ERR 2
+#define AOF_CRIT 3
```

Comment:
terminology: i don't like CRIT.
maybe `AOF_FAILED` is better?
or maybe we wanna have specific AOF_FORMAT_ERR, and AOF_INCOMPLETE_ERR, etc (even if we don't make that distinction in the caller)?

Reply:
relaced with AOF_FAILED